### PR TITLE
Feature M-1 -Order Model

### PR DESCRIPTION
**Pull request by madhu06492**
Created the model class for Order

### CODE REVIEW

File: src/main/java/com/example/model/Order.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
+package com.example.model;
+
```
Comment (RanaRanvijaySingh) [inline src/main/java/com/example/model/Order.java:2]: Unnecessary line, Please remove this.

File: src/main/java/com/example/model/Order.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    @Column(name="orderProducts")
+    private List<Product> products;
+    @Column(name="orderUser")
+
+    //change String to User data type
+    private String user;
+    //Change to  String to Address data type
+    @Column(name="orderAddress")
+    private String address;
+
```
Comment (RanaRanvijaySingh) [inline src/main/java/com/example/model/Order.java]: unnecessary blank lines.

File: src/main/java/com/example/model/Order.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    private Date date;
+    @Column(name="orderAmount")
+    private Double amount;
+    @Column(name="orderIsPaid")
+    private boolean isPaid;
+    @Column(name="orderProducts")
+    private List<Product> products;
+    @Column(name="orderUser")
+
+    //change String to User data type
```
Comment (RanaRanvijaySingh) [inline src/main/java/com/example/model/Order.java]: Add todo

File: src/main/java/com/example/model/Order.java (hunk 2 above)
Comment (madhu06492) [reply src/main/java/com/example/model/Order.java]: overActing

File: src/main/java/com/example/model/Order.java [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+@Table(name="order")
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@EqualsAndHashCode
+@ToString
+public class Order {
+    @Id
+    @GeneratedValue
+    @Column(name="orderId")
```
Comment (RanaRanvijaySingh) [inline src/main/java/com/example/model/Order.java]: Please change this to "id"

File: src/main/java/com/example/model/Order.java [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+    private long id;
+    @Column(name="orderDate")
```
Comment (RanaRanvijaySingh) [inline src/main/java/com/example/model/Order.java]: Please change this to "date"

File: src/main/java/com/example/model/Order.java [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+    @Column(name = "products")
+    private List<Product> products;
+    @Column(name = "user")
+    //TODO :change String to User data type
+    private String user;
+    //TODO: Change   String to UserAddress data type
+    @Column(name = "userAddress")
+    private String userAddress;
+    //TODO: change  string to orderAddress data type
+    @Column(name = "orderAddress")
```
Comment (RanaRanvijaySingh) [inline src/main/java/com/example/model/Order.java]: This should be martAddress

**Review by RanaRanvijaySingh — APPROVED** [review]

**Pull request merged**